legend-pure-core/legend-pure-m4/src/main/java/org/finos/legend/pure/m4/coreinstance/primitive/date/DateFormat.java

```diff
@@ -56,18 +61,18 @@ public static <T extends Appendable> T format(T appendable, String formatString,
                             timeZoneId.append(next);
                             escaped = false;
                         }
-                        else if (inQuotes)
+                        else if (inQuotes && next != '"')
```

Comment:
Perhaps we should just move the next == '"' case before this case?

Reply:
Fixed in commit 98bbe1bc1